cluster/manifests/skipper/deployment.yaml

```diff
@@ -429,7 +461,7 @@ spec:
         config/hash: {{"secret.yaml" | manifestHash}}
         kubernetes-log-watcher/scalyr-parser: |
           [{"container": "routesrv", "parser": "keyValue"}]
-        logging/destination: "{{.Cluster.ConfigItems.log_destination_local}}"
+        logging/destination: "{{ .ConfigItems.log_destination_local }}"
```

Comment:
Removed .Cluster prefix everywhere for consistency

Reply:
no please not!

Reply:
Why not? It should be either .ConfigItems or .Cluster.ConfigItems everywhere, I just picked .ConfigItems

Reply:
Let's keep the discussion in the other comment